switchboard/builtins.py

```diff
@@ -53,9 +55,10 @@ def get_field_value(self, instance, field_name):
         return super(IPAddressConditionSet, self).get_field_value(instance,
                                                                   field_name)
 
-    def get_group_label(self):
+    def get_group_label(self):  # pragma: nocover
```

Comment:
Why `nocover`?  You should be able to have a unit test instantiate this object and call this method.